samza-api/src/main/java/org/apache/samza/operators/windows/WindowKey.java

```diff
@@ -21,26 +21,35 @@
 /**
  * Key for a {@link WindowPane} emitted from a {@link Window}.
  *
- * @param <K> the type of the key in the incoming {@link org.apache.samza.operators.data.MessageEnvelope}.
+ * @param <K> the type of the key in the incoming message.
  *            Windows that are not keyed have a {@link Void} key type.
  *
  */
 public class WindowKey<K> {
```

Comment:
Since this is just merge from @vjagadish1989 's window/trigger patch, I will leave it to @vjagadish1989 to make the final call.